molgenis/molgenis-emx2
# chore: fixed example component hyperlinks to be valid

### What are the main changes you did
- Fixed hyperlinks in components webpage to show valid hyperlinks instead of ones that generate an error message.

### How to test
- Ensure no error messages are shown when visiting http://localhost:8080/apps/molgenis-components/#/component/FormInput (or PR-build equivalent).

### Checklist
